chpl/chpl-api/src/main/java/gov/healthit/chpl/ChplFF4JProvider.java

```diff
@@ -18,7 +16,9 @@ public ChplFF4JProvider() {
 
     @Override
     public FF4j getFF4j() {
+//        if (ff4j == null) {
```

Comment:
Should this be removed? Or does it need to be restored?

Reply:
Removed it.